python/src/LinearModelAlgorithm_doc.i.in

```diff
@@ -19,30 +19,65 @@ LinearModelResult
+The linear model can be estimated with or without a functional basis.
+
+If no basis is specified, the model is:
 
 .. math::
 
-    \tilde{Y} = \sum_{i=0}^p a_i \phi_i(X) + \epsilon
+    Y = a_0 + \sum_{i=1}^{n_x} a_i X_i + \epsilon
 
-where :math:`\epsilon` is the residual, supposed to follow the standard Normal
-distribution, :math:`\phi` a functional basis.
-The algorithm class enables to estimate the coefficients of the linear expansion.
+where :math:`\epsilon` is a random variable with zero mean and constant
+(and unknown) variance :math:`\sigma^2` and :math:`a_0, a_1, ..., a_{n_x} \in \Rset`
+are unknown coefficients.
+The algorithm estimates the coefficients
+:math:`a_0, a_1, ..., a_{n_x}` of the linear model.
+Moreover, the method estimates the variance :math:`\sigma^2`.
 
-If basis is not specified, the underlying model is :
+If a functional basis is specified, let :math:`p \in \Nset` be the number of
+functions in the basis.
+For :math:`j \in \{1, ..., p\}`, let :math:`\phi_j : \Rset^{n_x} \rightarrow \Rset`
+be the :math:`i`-th basis function.
+The linear model is:
 
 .. math::
 
-    \tilde{Y} = a_0 + \sum_{i=1}^n a_i X_i + \epsilon
+    Y = \sum_{j=1}^p a_j \phi_j(\vect{X}) + \epsilon
+
+where :math:`\epsilon` is a random variable with zero mean and constant
+(and unknown) variance :math:`\sigma^2` and :math:`a_1, ..., a_p \in \Rset`
+are unknown coefficients.
+The algorithm estimates the coefficients
+:math:`a_1, ..., a_p` of the linear model.
+Moreover, the method estimates the variance :math:`\sigma^2`.
+
+The coefficients :math:`a_i` are evaluated using a linear least squares method.
+Default method is `QR`. User might also choose `SVD` or `Cholesky` by
+setting the `LinearModelAlgorithm-DecompositionMethod` key of the :class:`~openturns.ResourceMap`.
+This choice can be based depending on the following parameters.
+
+- The Cholesky can be safely used if the functional basis is orthogonal,
+  because this ensures that the columns of the design matrix are
+  orthogonal, so that evaluating the Gram matrix does not increase
+  the condition number.
+- Selecting the decomposition method can also be based on the sample size.
+
+Please read the :meth:`~openturns.LeastSquaresMethod.Build` help page on this topic.
 
-The coefficients :math:`a_i` are evaluated using a least squares method.
-Default method is `QR`. User might also choose `SVD` or `Cholesky` (useful
-if basis is orthogonal) and large dataset.
+The :class:`~openturns.LinearModelAnalysis` class can be used for a detailed
+analysis of the linear model.
 
-The evaluation of the coefficients is completed by some useful parameters that could
-help the diagnostic of the linearity.
+No scaling is involved in this method.
+The scaling of the data, if any, is the responsibility of the user of the algorithm.
+This may be useful is, for example, we use a linear model (without functional basis)
+with very different input magnitudes and use the Cholesky decomposition.
```

Comment:
My knowledge is that any orthogonal matrix decomposition of the design matrix (such as QR or SVD) is stable regardless of the potential scaling of the data. Of course, using a raw data will be influential on the absolute value of the eigenvalues, but this will still provide a stable estimate of them. Have you any numerical experiments or bibliographic reference which states otherwise?

Reply:
I will ask for references. This is a kind of message I got in the past from linear algebra experts